examples/client.rs

```diff
@@ -1,14 +1,29 @@
+
+extern crate async_trait;
+extern crate dapr;
+use prost_types::Any;
+
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // Get the Dapr port and create a connection
-    let port: u16 = std::env::var("DAPR_GRPC_PORT")?.parse()?;
-    let addr = format!("https://127.0.0.1:{}", port);
+    //let port: u16 = std::env::var("DAPR_GRPC_PORT")?.parse()?;
```

Comment:
nit: shouldn't we uncomment it ?

Reply:
I tried to run this client using `dapr run --app-id rustapp --app-port 3000 --grpc-port 3800 cargo run --example client` so that we could use the `DAPR_GRPC_PORT` but it's throwing an error saying:
```bash
Error: unknown flag: --example
Usage:
  dapr run [flags]
...
```
So I believe we can't yet run application command with flag (--example in this case). That's the reason I hard coded port value.

Reply:
This issue is resolved by [using an extra `--` before app args](https://github.com/dapr/cli/issues/332#issuecomment-627648623). Please review.